qiskit/opflow/converters/z2_taper.py

```diff
@@ -0,0 +1,92 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+""" Z2 Symmetry Tapering Class """
+
+import logging
+from typing import List, Optional, Tuple, Union, cast
+
+from qiskit.quantum_info import Pauli
+
+from ..primitive_ops.pauli_sum_op import PauliSumOp
+from ..operator_base import OperatorBase
+from .converter_base import ConverterBase
+from .z2_symmetries import Z2Symmetries
+
+logger = logging.getLogger(__name__)
+
+
+class Z2Taper(ConverterBase):
+    """ Z2 Symmetries """
```

Comment:
Can you update the docstring here - I think this is to support the two qubit reduction for Chemistry we had in the past. There is text in the old method that perhaps can be re-used here